packages/video_player/ios/Classes/VideoPlayerPlugin.m

```diff
@@ -305,10 +305,6 @@ - (void)sendInitialized {
     if (height == CGSizeZero.height && width == CGSizeZero.width) {
       return;
     }
-    // The player may be initialized but still needs to determine the duration.
```

Comment:
I think the quicker fix can be something like
```objc
if ([self duration] == 0 && ![self isDurationIndefinate])
```

And create a method to check if the duration isDurationIndefinate as simple as 
```objc
return CMTIME_IS_INDEFINITE([[_player currentItem] duration]);
```

This way we can avoid the breakage also we won't need to introduce the new api for duration anymore.